libs/components/core/src/lib/modules/scrollable-host/scrollable-host.service.ts

```diff
@@ -138,6 +138,7 @@ export class SkyScrollableHostService {
           }
           scrollableHost = newScrollableHost;
           newScrollableHostObservable = new Subject();
+          /* istanbul ignore else */
           if (newScrollableHost) {
```

Comment:
Was this causing you issues? I was surprised to see this in this PR

Reply:
I took the change out of this PR. I was getting errors on less than 100% coverage on core.